packages/mask/src/components/CompositionDialog/useSubmit.ts

```diff
@@ -17,7 +17,7 @@ export function useSubmit(onClose: () => void, reason: 'timeline' | 'popup' | 'r
     return useCallback(
         async (info: SubmitComposition) => {
             const { content, encode, target } = info
-            const currentProfile = globalUIState.profiles.value?.[0].identifier
+            const currentProfile = globalUIState.profiles.value?.[0]?.identifier
```

Comment:
Why is the type checker not working here?

Reply:
You need either Array.prototype.at or enable --noUncheckedIndexedAccess flag

![image](https://user-images.githubusercontent.com/5390719/168639195-999190f8-299a-4032-b3c3-1530a94d3ec0.png)